sermant-agentcore/sermant-agentcore-implement/pom.xml

```diff
@@ -125,6 +126,26 @@
             <artifactId>asm-commons</artifactId>
             <version>${asm.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.envoyproxy.controlplane</groupId>
+            <artifactId>api</artifactId>
+            <version>${envoyproxy.controlplane.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty</artifactId>
+            <version>${grpc.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-stub</artifactId>
+            <version>${grpc.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-protobuf</artifactId>
+            <version>${grpc.version}</version>
+        </dependency>
```

Comment:
once you add new dependency, remember add to https://github.com/sermant-io/Sermant/tree/develop/LICENSES and https://github.com/sermant-io/Sermant/blob/develop/NOTICE. You could do it after you complete this feature

Reply:
I will do this job after i complete thie feature